### PR TITLE
Use equalizer with ROG Ally

### PR DESCRIPTION
**Pull request by game-de-it**
Apply the equalizer settings tuned by discord user "tomoniutaou33" to ROG ally.
This can solve the problem of low volume in ChimeraOS.

### CODE REVIEW

**Comment by pastaq** [comment]
This looks good. I will test it later when the sound patch is updated on unstable. Thank you!

File: usr/share/device-quirks/scripts/asus/ally/ally.sh [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  exit 1
+fi
+
+if [ -d /tmp/frzr_root ]; then
+  source ${MOUNT_PATH}/etc/device-quirks/device-quirks.conf
+else
+  source /etc/device-quirks/device-quirks.conf
+fi
+
+# Create an equalizer for pipwier.
```
Comment (philmmanjaro) [inline usr/share/device-quirks/scripts/asus/ally/ally.sh:13]: Small typo for pipewire here ...

Comment (game-de-it) [reply usr/share/device-quirks/scripts/asus/ally/ally.sh]: ah. . Embarrassingly I had a typo.:)
I have corrected the mistakes.
Thank you!

**Comment by game-de-it** [comment]
tomoniutaou33 tuned the sink-eq.conf file to fix bass distortion.

File: usr/share/device-quirks/scripts/asus/ally/ally.sh [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+if [ $(whoami) != 'root' ]; then
+  echo "You must be root to run this script."
+  exit 1
+fi
+
+if [ -d /tmp/frzr_root ]; then
+  source ${MOUNT_PATH}/etc/device-quirks/device-quirks.conf
+else
+  source /etc/device-quirks/device-quirks.conf
+fi
```
Comment (pastaq) [inline usr/share/device-quirks/scripts/asus/ally/ally.sh:11]: 7-11 are not necessary as the applies quirk does not invoke a firmware override

**Review by pastaq — CHANGES_REQUESTED** [review]
ally.sh

File: usr/share/device-quirks/scripts/asus/ally/ally.sh (hunk 2 above)
Comment (game-de-it) [reply usr/share/device-quirks/scripts/asus/ally/ally.sh:11]: Thank you for your advice!
Delete the appropriate line.

Comment (game-de-it) [reply usr/share/device-quirks/scripts/asus/ally/ally.sh]: Removed firmware override processing.

**Review by pastaq — APPROVED** [review]

**Pull request merged**